pages/en/lb4/Schemas.md

```diff
@@ -9,9 +9,115 @@ summary:
 ---
 {% include content/tbd.html %}
 
-> Stub page per [#554](https://github.com/strongloop/loopback-next/issues/554).
+## Overview
+A `schema` represents the definition of a model in LoopBack, with respect to
+the [datasource juggler](https://github.com/strongloop/loopback-datasource-juggler).
+Currently, we provide the `@loopback/repository` module, which provides special
+decorators for adding metadata to your TypeScript/JavaScript
+classes in order to use them with the legacy implementation of the Juggler.
+
+## Using Legacy Juggler
+Defining a schema for use with the legacy juggler involves decorating your
+classes with the `@model` and `@property` decorators.
+
+```ts
+import { model, property } from '@loopback/repository';
+
+@model()
+export class Product extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+    description: 'The unique identifier for a product',
+  })
+  id: number;
+
+  @property({type: 'string'})
+  name: string;
+
+  @property({type: 'string'})
+  slug: string;
+
+  constructor(data?: Partial<Product>) {
+    super(data);
+  }
+}
+```
+
+### Model Decorator
+The model decorator can be used without any additional parameters, or can take
+in a ModelDefinitionSyntax object which follows the general format provided in
+LoopBack 3:
+```ts
+{
+@model({
+  name: "Foo",
+  properties: {
+    // define properties here.
+  },
+  settings: {
+    // etc...
+  }
+})
+class Foo {
```

Comment:
Since you're showing both ways of using the `@model` decorator, wouldn't it be 
```ts
model()
+export class Foo extends Entity {
  // etc...
}
```
here?

Reply:
I believe it's important for `Foo` to extend `Entity`, otherwise the model cannot be used with the juggler bridge.

Can we use a "real" model name like "Category" instead of "Foo" please?

As for `model()` usage, that's already documented in the previous section.